float64 x_position
float64 y_position
float64 count_rate